tests/mock_crust_tests.rs

```diff
@@ -40,6 +40,10 @@
 extern crate itertools;
 #[macro_use]
 extern crate log;
+#[cfg_attr(feature="clippy", allow(useless_attribute))]
+#[allow(unused_extern_crates)]
+#[macro_use]
+extern crate maidsafe_utilities;
```

Comment:
I agree, but I added it since [I used `unwrap!()`](https://github.com/maidsafe/routing/pull/1254/files#diff-12e885abc7235cb34d3e197fb87dbb2aR37).